pages/guides/_meta.json

```diff
@@ -1,6 +1,4 @@
 {
   "how_to_uncross_orderbook": "How to uncross the orderbook",
-  "how_to_send_usdc_to_dydx": "How to send USDC from Ethereum to dYdX",
   "how_to_interpret_block_data_for_trades": "How to interpret block data for trades",
-  "orderbook_stream": "Orderbook Stream"
 }
```

Comment:
Is this missing a couple of docs?
- setting up raspberry pi...
- how to send usdc